mobile/lib/screens/settings/settings_page.dart

```diff
@@ -1,31 +1,61 @@
+import 'dart:io';
+
 import 'package:app/blocs/blocs.dart';
+import 'package:app/constants/constants.dart';
 import 'package:app/models/models.dart';
 import 'package:app/services/services.dart';
 import 'package:app/themes/theme.dart';
+import 'package:app/utils/utils.dart';
 import 'package:app/widgets/widgets.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:permission_handler/permission_handler.dart';
 
 import '../auth/auth_verification.dart';
 import '../feedback/feedback_page.dart';
 import 'about_page.dart';
-import 'settings_page_widgets.dart';
 
 class SettingsPage extends StatefulWidget {
   const SettingsPage({super.key});
-
+  // TODO implement restoration for android permissions
   @override
   State<SettingsPage> createState() => _SettingsPageState();
 }
 
-class _SettingsPageState extends State<SettingsPage> {
-  late BuildContext _loadingContext;
+class _SettingsPageState extends State<SettingsPage>
+    with WidgetsBindingObserver {
+  late String enableLocationMessage;
+  late String enableNotificationsMessage;
+  late String disableLocationMessage;
+  late String disableNotificationsMessage;
 
   @override
   void initState() {
     super.initState();
-    _loadingContext = context;
+    WidgetsBinding.instance.addObserver(this);
+    context.read<SettingsBloc>().add(const InitializeSettings());
+    if (Platform.isAndroid) {
+      enableLocationMessage =
+          'To turn on location, go to\nApp Info > Permissions > Location > Allow only while using the app';
+      enableNotificationsMessage =
+          'To turn on notifications, go to\nApp Info > Notifications';
+
+      disableLocationMessage =
+          'To turn off location, go to\nApp Info > Permissions > Location > Deny';
+      disableNotificationsMessage =
+          'To turn off notifications, go to\nApp Info > Notifications';
+    } else {
+      enableLocationMessage =
+          'To turn on location, go to\nSettings > AirQo > Location > Always';
+      enableNotificationsMessage =
+          'To turn on notifications, go to\nSettings > AirQo > Notifications';
+
+      disableLocationMessage =
+          'To turn off location, go to\nSettings > AirQo > Location > Never';
+      disableNotificationsMessage =
+          'To turn off notifications, go to\nSettings > AirQo > Notifications';
+    }
```

Comment:
Instructions on how to enable/disable notifications and location from the phone settings.